sources/packages/web/src/services/http/ConfirmationOfEnrollmentApi.ts

```diff
@@ -46,4 +46,20 @@ export class ConfirmationOfEnrollmentApi extends HttpBaseClient {
       throw error;
     }
   }
+
+  public async rollbackCOE(
+    locationId: number,
+    applicationId: number,
+  ): Promise<void> {
+    try {
+      await this.apiClient.post(
```

Comment:
is it possible to have a GET instead of a POST? it does not have a payload, so wanted to check, is there a reason to have it as post call?

Reply:
Cool i get it now, thanks